### ISSUE TITLE
[BUG]: Add horizontalPadding to HelpDialog items

### ISSUE THREAD

**Issue by SanjaySargam** [ankidroid/Anki-Android#14042]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. Go to NavigationDrawer
2. Click on Help

### Expected behaviour

To look better, there must be some horizontalPadding to items

### Actual behaviour

No horizontalPadding to HelpDialog items

![WhatsApp Image 2023-06-28 at 12 05 27 AM](https://github.com/ankidroid/Anki-Android/assets/65113071/ba043563-4bfc-44be-80e4-a9048604f6b1)

### Debug info

```text
AnkiDroid Version = 2.16.0beta4-debug

Android Version = 13

ProductFlavor = amazon

Manufacturer = realme

Model = RMX3521

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; RMX3521 Build/RKQ1.211119.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/114.0.5735.130 Mobile Safari/537.36

ACRA UUID = 20b2dc80-f22d-4759-9817-396d281f95b3

New schema = false

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by SanjaySargam** [comment]
Working on it

**Comment by pratyaksh1610** [comment]
@SanjaySargam Can I work on this , if not addressed yet ?

**Comment by SanjaySargam** [comment]
> @SanjaySargam Can I work on this , if not addressed yet ?

I'm sorry @pratyaksh1610 . I already started implementing.

**Comment by dae** [comment]
Should be fixed when #14071 lands
